nixos/modules/services/databases/victoriametrics.nix

```diff
@@ -31,7 +31,11 @@ let
 in
 {
   options.services.victoriametrics = {
-    enable = mkEnableOption "VictoriaMetrics is a fast, cost-effective and scalable monitoring solution and time series database.";
+    enable = mkEnableOption ''
+      Whether to enable VictoriaMetrics in single-node mode.
```

Comment:
`mkEnableOption` generates a description `Whether to enable <argument of mkEnableOption>.`, so this will look wrong in the manual.

I think at that point it's absolutely fine to use `mkOption`, whether or not two lines are saved is really not a big deal I'd argue.